### PR TITLE
Fix#80

### PR DESCRIPTION
**Pull request by AlteredCoder**
- make `cscli backup` restore and save plugins configuration file
- make `wizard.sh --binupgrade` upgrade also plugins
- make `wizard.sh --bininstall` install plugins

### CODE REVIEW

File: cmd/crowdsec-cli/backup-restore.go [hunk 1]
```diff
@@ -195,6 +195,36 @@ func restoreFromDirectory(source string) error {
 	}
 	log.Infof("Restore acquis to %s", yamlAcquisFile)
 
+	/* Restore plugins configuration */
+	var pluginsConfigFile []string
+	err = filepath.Walk(fmt.Sprintf("%s/plugins/backend/", source), func(path string, info os.FileInfo, err error) error {
```
[the remark this answers is not in the document]
Comment (AlteredCoder) [reply cmd/crowdsec-cli/backup-restore.go:198]: i think linter is crazy :p .
err is checked [here](https://github.com/crowdsecurity/crowdsec/blob/616503058735b647bbd904ba30da230001d5a575/cmd/crowdsec-cli/backup-restore.go#L211)

File: cmd/crowdsec-cli/backup-restore.go [hunk 2]
```diff
@@ -367,6 +397,37 @@ func backupToDirectory(target string) error {
 		return fmt.Errorf("unable to write credentials to %s : %s", apiCredsDumped, err)
 	}
 	log.Infof("Saved configuration to %s", target)
+
+	/* Backup plugins configuration */
+	var pluginsConfigFile []string
+	err = filepath.Walk(outputCTX.Config.BackendFolder, func(path string, info os.FileInfo, err error) error {
```
[the remark this answers is not in the document]
Comment (AlteredCoder) [reply cmd/crowdsec-cli/backup-restore.go:401]: same here, err is checked

File: cmd/crowdsec-cli/backup-restore.go [hunk 3]
```diff
@@ -195,6 +193,36 @@ func restoreFromDirectory(source string) error {
 	}
 	log.Infof("Restore acquis to %s", yamlAcquisFile)
 
+	/* Restore plugins configuration */
+	var pluginsConfigFile []string
+	walkErr := filepath.Walk(fmt.Sprintf("%s/plugins/backend/", source), func(path string, info os.FileInfo, err error) error {
+		fi, err := os.Stat(path)
```
Comment (buixor) [inline cmd/crowdsec-cli/backup-restore.go:199]: I think you're overwritting the `err` that is given as a parameter

File: cmd/crowdsec-cli/backup-restore.go [hunk 4]
```diff
@@ -367,6 +395,37 @@ func backupToDirectory(target string) error {
 		return fmt.Errorf("unable to write credentials to %s : %s", apiCredsDumped, err)
 	}
 	log.Infof("Saved configuration to %s", target)
+
+	/* Backup plugins configuration */
+	var pluginsConfigFile []string
+	walkErr := filepath.Walk(outputCTX.Config.BackendFolder, func(path string, info os.FileInfo, err error) error {
```
Comment (buixor) [inline cmd/crowdsec-cli/backup-restore.go:401]: I think you're overwritting the err that is given as a parameter

**Pull request merged**